### DISCUSSION TITLE
Quarto website on Github pages w/ custom domain not updating!

### DISCUSSION THREAD

**Discussion by SowdaminiSP** [quarto-dev/quarto-cli#7678] [Q&A] [answered]
### Description

Hi Quarto community,
I just wanted to check if anyone else experienced similar issues and managed to fix it?:) 
The qmd files render locally and are updated in the docs folder in my repo. feel free to have a look: https://github.com/Sustainable-Aquafeeds-Project/Sustainable-Aquafeeds-Project.github.io
However, the website linked to the docs folder is not updating. 
Any input on this is greatly appreciated! 

Thank you,
A quarto newbie :)

**Comment by mcanouil** [comment]
What is not updated exactly?
Your last commit is from
2 hours ago and the last GitHub Pages deployment a bit after.
So it should be up to date.

https://sustainable-aquafeeds-project.github.io/research/spillias-et-al2022.html

Now you are talking about custom domain but I don't see anything in your repository.

**Comment by SowdaminiSP** [comment, marked as answer]
Hi Mickaël,
The issue was that the website was not automatically updating (maybe it still doesn't) when I made changes to it on quarto and pushed it to the repo. I noticed this around a month ago.
unpublishing --> rerunning most recent workflow --> adding custom domain name seemed to have fixed the issue now. I will reopen this discussion if any future changes aren't automatically updated on the website :)

**Reply by mcanouil** [reply]
You have a "deployment" section on the right of your repository.
This tells you when your website is deployed to GitHub Pages.
It is not enough for your website to be in your repository.
The deployment does not necessarily occurs on push, so you have to wait.

Note that you could speed up a bit deployment by adding `.nojekyll` alongside the rendered files.
This will tell GitHub to not try to use Jekyll and skip this test from the deployment workflow.

**Comment by SowdaminiSP** [comment]
Yes, of course. I did track the deployment page as well. Most recent deployments (before re-running workflow) did not reflect the changes made in the repo.
Any idea why this would’ve happened?
________________________________

**Reply by mcanouil** [reply]
Here without logs or anything to reproduce, it will only be a very wild guess. 🤷‍♂️

**Comment by SowdaminiSP** [comment]
I'll get in touch with you if/when it happens next time! Hopefully it doesn't. Thanks for your input!
________________________________
